recipes-app/iot2050setup/files/iot2050setup.py

```diff
@@ -275,6 +255,177 @@ def getPullMode(self, index):
         elif self.checkPullModeConfig('Hiz', index):
             return 'Hiz'
 
+
+class ExternalSerialMode(BoardConfigurationUtility):
+    def __init__(self, topmenu):
+        self.topmenu = topmenu
+        super().__init__()
```

Comment:
If topmenu is shared then put it into the base class.

Reply:
this is not from base class